mantidproject/mantid
# Remove debug poco lib packages from equivs file

**Description of work.**

Removed poco lib debug packages from the equivs control file so that it works on Ubuntu 18.04 as well as 16.04. Fixed problem occuring if `PocoFoundation` debug library is found by CMake but one of the others is missing.

**To test:**

On Ubuntu check this successfully creates the dev dependency package
```
equivs-build mantid/buildconfig/dev-packages/deb/mantid-developer/ns-control
```
Review CMake change, could also check behaviour if not all debug libraries are installed on the system. 

Fixes #23048

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
